plugins/shrs_completion/Cargo.toml

```diff
@@ -19,3 +19,5 @@ dirs = { version = "5.0" }
 
 thiserror = "1"
 anyhow = "1"
+rhai = { version = "1.16" }
```

Comment:
hm I'm not sure if we should do this stuff in `shrs_completion` crate. This was meant as a library for users to install to get a bunch of completions out of the box. Maybe create new plugin called `shrs_rhai_completion` for this.